core/provisioning-java/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
Add new line at the end of this file, to comply with checkstyle.

Reply:
Done